src/actions/__tests__/evaluation-queue.test.js

```diff
@@ -0,0 +1,71 @@
+import { getEvaluationResolvers, resolveEvaluation, rejectEvaluation,
```

Comment:
Testing this is tricky and I think I should probably refactor the API around the queue to make the tests feel a little cleaner.

Reply:
The latest commits refactor the whole thing, and the tests now feel nicer & the API a bit cleaner.